tests/wicked/startandstop/ref/t11_vlan_ifdown_modify_one_config.pm

```diff
@@ -0,0 +1,25 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: VLAN - ifdown, modify one config, ifreload, ifdown, ifup
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use base 'wickedbase';
+use strict;
+use testapi;
+
+
+sub run {
+    my ($self) = @_;
+    record_info('Info', 'VLAN - ifdown, modify one config, ifreload, ifdown, ifup');
```

Comment:
Do you plan to add some tests here or this module only serves the purpose of logging the steps?

Reply:
in this case yes , just added. but generally we have a lot of 'empty' files like this containing just `record_info` message. We decided to do so to have same flow to all wicked tests. You think we should change this ?

Reply:
idea behind that each test use two nodes , one is SUT and second is REF. Main activity always happening in SUT and sometimes REF is not acting at all during whole module. But "optimization" of such modules would force us adding some complex logic in other places so it is easier just to keep empty module instead

Reply:
We use this as the full title of the test, taken from Wicked original test repo.
Since the name of the module is too short to understand, we decided to put it as record_info.